code/game/machinery/rechargestation.dm

```diff
@@ -66,7 +66,7 @@
 	recharge_speed = 0
 	repairs = 0
 	for(var/datum/stock_part/capacitor/capacitor in component_parts)
-		recharge_speed += (capacitor.tier * STANDARD_CELL_CHARGE * 0.01)
+		recharge_speed += (capacitor.tier * STANDARD_CELL_CHARGE * 0.1)
```

Comment:
Right so my explanation in https://github.com/tgstation/tgstation/pull/82191#issuecomment-2016706147 was wrong. I had assumed the previous value of `STANDARD_CELL_CHARGE` was 10000 but after looking at https://github.com/tgstation/tgstation/pull/82189#discussion_r1536737640 i now understand it was 1000.

so 100 / 1000 = 0.1 which is the true ratio